components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -982,8 +989,10 @@ void ConversationDriver::SendFeedback(
       },
       std::move(callback));
 
-  feedback_api_->SendFeedback(category, feedback, rating_id,
-                              std::move(on_complete));
+  feedback_api_->SendFeedback(
+      category, feedback, rating_id,
+      send_page_url ? std::optional<GURL>(GetPageURL()) : std::nullopt,
```

Comment:
I think I would prefer it if we only provided the hostname to the FeedbackAPI here so that we put the restrictive logic all in this class.

Reply:
ok, i have changed this, `SendFeedback` takes an optional string hostname now.